workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/ScreenOverlayDialogFactory.kt

```diff
@@ -39,12 +41,12 @@ public open class ScreenOverlayDialogFactory<S : Screen, O : ScreenOverlay<S>>(
     val modal = initialRendering is ModalOverlay
 
     return OverlayDialogHolder(initialEnvironment, dialog) { overlayRendering, environment ->
-      // For a modal, on each update put a no-op backPressedHandler in place on the
+      // For a modal, on each update put a no-op backHandler in place on the
       // decorView before updating, to ensure that the global androidx
       // OnBackPressedDispatcher doesn't fire any set by lower layers. We put this
       // in place before each call to show(), so the real content view will be able
       // to clobber it.
-      if (modal) content.view.backPressedHandler = {}
+      if (modal) content.view.setBackHandler {}
```

Comment:
Given all the no-op handlers we use, having a default of `{}` might be nice. Although maybe that's just in these internal use cases?